index.js

```diff
@@ -0,0 +1,57 @@
+const redBlock = document.querySelector('.block--red');
+const blueBlock = document.querySelector('.block--blue');
+const greenBlock = document.querySelector('.block--green');
+const pinkBlock = document.querySelector('.block--pink');
+const grayBlock = document.querySelector('.block--gray');
+
+// -------------Rockets commit-------------
+/*
+let count = 0;
+
+const blockTop = (event) => {
+    event.target.style.order = count--;
+}
+
+blueBlock.addEventListener('click', blockTop);
+redBlock.addEventListener('click', blockTop);
+greenBlock.addEventListener('click', blockTop);
+pinkBlock.addEventListener('click', blockTop);
+grayBlock.addEventListener('click', blockTop);
+*/
+
+// ----------Travelers commit-----------
+
+// I tried to make the code more compact and loop through however it broke each time. I left the attempts commented
+
+// const blocks = document.querySelector('.blocks').childNodes; //tried a few different ways to select them here
```

Comment:
I would recommend that you do a console.log(blocks); to see what is being called. Actually, just console.log EVERYWHERE!